### PR TITLE
Trivial changes

### PR DESCRIPTION
**Pull request by theuni**
It's been a while now, so here's what's piled up in trivial.

Fwiw, since we dropped the policy on header include order, I'll start NACKing those changes in trivial from this point forward, unless they're necessary for some specific reason. With the current modularization effort that constantly touches includes, they just cause unnecessary merge conflicts.

I'll also be more picky about arbitrary cosmetic changes, as they could ping/pong forever.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -3479,7 +3479,7 @@ void static ProcessGetData(CNode* pfrom)
                         send = mi->second->IsValid(BLOCK_VALID_SCRIPTS) && (pindexBestHeader != NULL) &&
                             (mi->second->GetBlockTime() > pindexBestHeader->GetBlockTime() - 30 * 24 * 60 * 60);
                         if (!send) {
-                            LogPrintf("ProcessGetData(): ignoring request from peer=%i for old block that isn't in the main chain\n", pfrom->GetId());
+                            LogPrintf("%s: ignoring request from peer=%i  for old block that isn't in the main chain\n", __func__, pfrom->GetId());
```
Comment (paveljanik) [inline src/main.cpp]: one new space before "for old block".

File: src/chainparams.cpp [hunk 2]
```diff
@@ -151,11 +151,11 @@ class CMainParams : public CChainParams {
         assert(consensus.hashGenesisBlock == uint256S("0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"));
         assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
 
-        vSeeds.push_back(CDNSSeedData("bitcoin.sipa.be", "seed.bitcoin.sipa.be"));
-        vSeeds.push_back(CDNSSeedData("bluematt.me", "dnsseed.bluematt.me"));
-        vSeeds.push_back(CDNSSeedData("dashjr.org", "dnsseed.bitcoin.dashjr.org"));
-        vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com"));
-        vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org"));
+        vSeeds.push_back(CDNSSeedData("bitcoin.sipa.be", "seed.bitcoin.sipa.be")); // Peiter Wiulle
```
Comment (Diapolo) [inline src/chainparams.cpp]: This seems to be not THE Pieter Wuille, we all know ;)?

Comment (sipa) [inline src/chainparams.cpp]: Who is that guy?!!?

File: src/chainparams.cpp [hunk 3]
```diff
@@ -151,11 +151,11 @@ class CMainParams : public CChainParams {
...
+        vSeeds.push_back(CDNSSeedData("bluematt.me", "dnsseed.bluematt.me")); // Matt Corallo
+        vSeeds.push_back(CDNSSeedData("dashjr.org", "dnsseed.bitcoin.dashjr.org")); // Luke Dashjr
+        vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com")); // Addy Yeow
```
Comment (ayeowch) [inline src/chainparams.cpp]: Christian Decker operates seed.bitcoinstats.com

**Comment by jgarzik** [comment]
ACK

**Comment by laanwj** [comment]
> I'll also be more picky about arbitrary cosmetic changes, as they could ping/pong forever

Indeed.

**Comment by laanwj** [comment]
ACK, apart from @sipa's name

**Comment by paveljanik** [comment]
ACK (nits: @sipa's name, one additional space)

File: src/chainparams.cpp (hunk 2 above)
Comment (Diapolo) [inline src/chainparams.cpp]: @theuni https://github.com/theuni/bitcoin/pull/17
You just need to rebase to current master I guess...

**Comment by laanwj** [comment]
Merged via 6a0bcb80e4ce380a39f81b4fb01150d5b7e6cdf7, after fixing nits.

**Comment by ayeowch** [comment]
@laanwj Looks like one nit is missing in the merged commit: seed.bitcoinstats.com is operated by Christian Decker.
https://github.com/bitcoin/bitcoin/commit/6a0bcb80e4ce380a39f81b4fb01150d5b7e6cdf7#diff-64cbe1ad5465e13bc59ee8bb6f3de2e7R157

**Pull request closed without merging**